docs/modules/reporting.md

```
@@ -18,7 +18,24 @@ script ([ndb-setup](https://github.com/Aam-Digital/ndb-setup)) also)_
 1. Set up necessary environment variables (e.g. using an `application.env` file for docker compose under
    `config/aam-backend-service/application.env` from the root folder where the "docker-compose.yml" exists):
     - see [example .env](/templates/aam-backend-service/application.template.env)
-    - CRYPTO_CONFIGURATION_SECRET: _a random secret used to encrypt data_
+    - `FEATURES_REPORTING_ENABLED=true` — enables this module (default `true` in the template)
+    - `CRYPTO_CONFIGURATION_SECRET` — a random secret used to encrypt data
+
+### Check if feature is enabled
+
+You can make a request to the API to check if the reporting feature is currently enabled and available:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)